From 4030b4204deac3f8ab0d560bff2d8ae692489558 Mon Sep 17 00:00:00 2001
From: Ivan Mironov <mironov.ivan@gmail.com>
Date: Sun, 1 Nov 2020 16:09:40 +0500
Subject: [PATCH] Read environment from /etc/sysconfig/irqbalance

---
 misc/irqbalance.service | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/misc/irqbalance.service b/misc/irqbalance.service
index e7a33361bd7a..5700d4c40002 100644
--- a/misc/irqbalance.service
+++ b/misc/irqbalance.service
@@ -6,7 +6,7 @@ ConditionVirtualization=!container
 
 [Service]
 EnvironmentFile=-/usr/lib/irqbalance/defaults.env
-EnvironmentFile=-/path/to/irqbalance.env
+EnvironmentFile=/etc/sysconfig/irqbalance
 ExecStart=/usr/sbin/irqbalance --foreground $IRQBALANCE_ARGS
 CapabilityBoundingSet=
 NoNewPrivileges=yes
-- 
2.28.0

